### ISSUE TITLE
What type of data should be listed in the Dataset Examples?

### ISSUE THREAD

**Issue by dorahermes** [INCF/BIDS-Starter-Kit#90]
I was just wondering what the type of data should go in Dataset Examples: should Raw (Non-BIDS) data be in there? The term raw is a bit confusing since BIDS is meant for raw data.

@Park-Patrick @KirstieJane

**Comment by emdupre** [comment]
Non-BIDS data could go in a `source` data folder, maybe ?

**Comment by chrisfilo** [comment]
The folder in the spec for non-harmonized data is `sourcedata/`

**Comment by emdupre** [comment]
Yes, sorry, I mis-typed. Was that what you were referring to, though, @dorahermes ?

**Comment by dorahermes** [comment]
Thank you! Yes, that is what I meant. 

In that case, is the following source data?
- Brainhack Western 2018 source dataset: ds001
for this BIDS compliant dataset:
- Brainhack Western 2018 dataset: ds001_BIDS

And what are these?
- Datasets used in Stanford Reproducibility Center tutorials: NKI Rockland
- Resting state fMRI datasets: FCON 1000
Are we intending to list open source neuroscience datasets? Then there is a long way to go... If we stick to BIDS compliant (and its sourcedata) it remains manageable and more relevant to the starters kit.

@emdupre @KirstieJane

**Comment by dorahermes** [comment]
Thank you! All sounds good, I updated this page to clarify. For now, I did not remove the links to non-BIDS data in there, but I think a focus on sourcedata --> BIDS is good!

**Comment by KirstieJane** [comment]
Brill! Thank you!

I’m on my phone now so leave this issue open and I’ll have a quick read through this afternoon and close if it all looks good :sparkles:
